libnavigation-core/build.gradle

```diff
@@ -54,6 +54,9 @@ dependencies {
     testImplementation project(':libtesting-utils')
     apply from: "${rootDir}/gradle/unit-testing-dependencies.gradle"
     testImplementation dependenciesList.commonsIO
+
+    testImplementation project(':libdirections-hybrid')
+    testImplementation project(':libtrip-notification')
```

Comment:
For the posterity, this is needed because looks like tests run with AS do not pick up `runtimeOnly` dependencies https://github.com/mapbox/mapbox-navigation-android/blob/2f3645d821dc11f187ae369e9cc79cc0710c997d/libnavigation-core/build.gradle#L42-L43 From CLI works though.

Reply:
That's true, but there are no issues with the tests in the `libdirections-hybrid`, which use the same setup :thinking:

Reply:
Well `MapboxHybridRouterTest` uses the `RobolectricTestRunner` (not sure if this could affect somehow) and also `MapboxNavigation` has all the `NavigationComponentProvider` setup (my guess is that there's something going on here). But in any case there are substantial differences between `MapboxHybridRouterTest` and `MapboxNavigationTest`. We might need to dig deeper, agreed 👍